src/index.html

```diff
@@ -10,7 +10,51 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="body">
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+      class="cover"
+    >
+      <label for="bigForm" class="form bigForm">
+        <img
+         src="images/Search.svg"
+         alt="Search icon"
+         class="icon iconBig"
```

Comment:
```suggestion
         class="icon"
```
leave just class `icon` and move styles to this class